src/tiled/selectsametiletool.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+ * selectsametiletool.cpp
+ * Copyright 2009-2010, Jeff Bland <jksb@member.fsf.org>
+ * Copyright 2010, Thorbjørn Lindeijer <thorbjorn@lindeijer.nl>
+ * Copyright 2010, Jared Adams <jaxad0127@gmail.com>
+ * Copyright 2011, Stefan Beller <stefanbeller@googlemail.com>
```

Comment:
You can give yourself some credit here. You can remove all the other names (also in the .h file).